src/components/RepoBadge/RepoBadge.js

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo } from 'react'
+import { blockExplorerUrl, Link } from '@aragon/ui'
+import { network } from '../../environment'
+import AppIcon from '../../components/AppIcon/AppIcon'
+import { KNOWN_ICONS, isKnownRepo } from '../../repo-utils'
+import { repoBaseUrl, parseHub } from '../../url-utils'
+
+function useRepoBadge(repo) {
+  const {
+    content: { appId, contractAddresses, name, icons },
+    version,
+  } = repo
+
+  return {
+    baseUrl: repoBaseUrl(repo),
+    icons: icons,
+    contractAddress: contractAddresses,
+    knownIcon: KNOWN_ICONS.has(appId) ? KNOWN_ICONS.get(appId) : null,
+    name: name,
+  }
+}
+
+const RepoBadge = ({ repo }) => {
+  const { baseUrl, icons, contractAddress, knownIcon, name } = useRepoBadge(
+    repo
+  )
+
+  return (
+    <div
+      css={`
+        display: inline-flex;
+        align-items: center;
+        min-width: 0;
+      `}
+    >
+      <Link
+        target="_blank"
+        href={blockExplorerUrl('address', contractAddress, {
+          networkType: network.type,
+        })}
```

Comment:
We may also want to reduce the focus ring since it's a bit large at the moment:

<img width="147" alt="Screen Shot 2019-08-26 at 8 18 21 PM" src="https://user-images.githubusercontent.com/4166642/63712781-abf50580-c83e-11e9-8f8f-bcbe5dc7e7cf.png">

Reply:
@bpierre how can this be done?

Reply:
We should probably not use `Link` here, but [`LinkBase`](https://github.com/aragon/aragon-ui/blob/newstyle/src/components/LinkBase/LinkBase.js) instead.

Reply:
Hmm, I don't see a difference with `LinkBase`:
![image](https://user-images.githubusercontent.com/3072458/63767981-5288e700-c8cf-11e9-9c79-76aff96bafe3.png)

Do I've to manually set the margins?

Reply:
Yes, you can set `focusRingSpacing=0` :-)

https://github.com/aragon/aragon-ui/blob/newstyle/src/components/LinkBase/LinkBase.js#L11